lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.spec.ts

```diff
@@ -87,34 +87,39 @@ describe('SearchChipAutocompleteInputComponent', () => {
         const optionsChangedSpy = spyOn(component.optionsChanged, 'emit');
         enterNewInputValue('op');
         await fixture.whenStable();
-
+        fixture.detectChanges();
         const matOptions = document.querySelectorAll('mat-option');
         expect(matOptions.length).toBe(2);
 
         const optionToClick = matOptions[0] as HTMLElement;
         optionToClick.click();
 
-        expect(optionsChangedSpy).toHaveBeenCalledOnceWith(['option1']);
-        expect(component.selectedOptions).toEqual(['option1']);
+        expect(optionsChangedSpy).toHaveBeenCalledOnceWith([{value: 'option1'}]);
+        expect(component.selectedOptions).toEqual([{value: 'option1'}]);
         expect(getChipList().length).toBe(1);
     });
 
     it('should apply class to already selected options', async () => {
         addNewOption('option1');
         enterNewInputValue('op');
 
-        const addedOptions = fixture.debugElement.queryAll(By.css('.adf-autocomplete-added-option'));
-
         await fixture.whenStable();
+        fixture.detectChanges();
+        const addedOptions = fixture.debugElement.queryAll(By.css('.adf-autocomplete-added-option'));
 
         expect(addedOptions[0]).toBeTruthy();
         expect(addedOptions.length).toBe(1);
     });
 
-    it('should limit autocomplete list to 15 values max', () => {
-        component.autocompleteOptions = ['a1','a2','a3','a4','a5','a6','a7','a8','a9','a10','a11','a12','a13','a14','a15','a16'];
+    it('should limit autocomplete list to 15 values max', async () => {
+        for (let i = 0; i < 17; i++) {
```

Comment:
small thing but probably you can even use: 
component.autocompleteOptions = Array.from({length: 10}, (el, i) => ({
  value: `a${i}`
}))

You don't need then write normal loop and maintain index manually.